.env.ci

```
@@ -1,8 +1,8 @@
-APP_NAME=interns2025c
+APP_NAME=LapGo
 APP_ENV=testing
 APP_KEY=base64:sCsJw8z+d/4ymp0OvzSip2h4Vp2hZZhpV2uOxgTqP94=
 APP_DEBUG=false
-APP_URL=http://interns2025c.blumilk.local.env
+APP_URL=http://LapGo.blumilk.local.env
```

```suggestion
APP_URL=http://lapgo.blumilk.local.env
```
Domain is not case sensitive.